recipes/testresources/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/73cd2eed94c576213c5f25ab57adf6d8c83e792a/schema.json
+schema_version: 1
+
+context:
+  name: testresources
+  version: 2.0.1
+  python_min: 3.7.0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/testresources-${{ version }}.tar.gz
+  sha256: ee9d1982154a1e212d4e4bac6b610800bfb558e4fb853572a827bc14a96e4417
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pbr
+    - pip
+    - setuptools
+  run:
+    - python >=${{ python_min }}
+    - pbr
+
+tests:
+  - python:
+      imports:
+        - testresources
+      pip_check: true
```

Comment:
```suggestion
  - python:
      python_version: ${{ python_min }}.*
      pip_check: true
      imports:
        - testresources
```

Reply:
If the python_version is not specified what version will be selected by default?
Just wondering.